### DISCUSSION TITLE
Easy(?) option for extra control inputs?

### DISCUSSION THREAD

**Discussion by pinkmouse101** [probonopd/MiniDexed#334] [Ideas]
Just a thought, as a more of a hardware hacker than a coder, could it be possible to use one of those generic USB numeric keyboards to get more menu buttons? Something like this:

https://www.amazon.co.uk/Targus-Numeric-Desktop-Computer-AKP10EU/dp/B002Q4Y4WS/ref=sr_1_4?crid=LTOU51EJDEKN&keywords=usb+numeric+keypad&qid=1660597519&sprefix=usb+numeric+keypad%2Caps%2C57&sr=8-4

USB keyboard support is obviously already implemented as a way to play notes, so could it be expanded as a function controller system? 

Or a USB mouse? Rip out the PCB, ignore the movement tracking, and just use the two buttons and clickable mouse wheel for navigation?

Feel free to tell me how stupid this idea is, I'm really just thinking aloud after a long day at work, and a rather large Jack Daniels... :-D

**Comment by probonopd** [comment]
Sure, could be done, but most HID (text/numbers) keyboards don't play well with "low-speed" USB which we seem to need for some (most?) MIDI controllers (music keyboards).
Maybe attaching a couple of buttons to a Raspberry Pi Pico that would act as a MIDI controller (and send MIDI messages when keys are pressed/controls are operated) would be the easiest way to do this.
